From 9602d0b07f4b6346deb6bd5bb90904385c407686 Mon Sep 17 00:00:00 2001
From: Gerald Baeza <gerald.baeza@st.com>
Date: Tue, 15 Dec 2020 15:51:16 +0100
Subject: [PATCH] ARM: dts: stm32: add ddrperfm on stm32mp151

The DDRPERFM is the DDR Performance Monitor embedded in STM32MP1 SOC.

Signed-off-by: Gerald Baeza <gerald.baeza@st.com>
Signed-off-by: Fabien Dessenne <fabien.dessenne@foss.st.com>
Change-Id: I706ebadb089a2f701b3c0748a8850fcea309cbe7
---
 arch/arm/boot/dts/st/stm32mp151.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/st/stm32mp151.dtsi b/arch/arm/boot/dts/st/stm32mp151.dtsi
index 89d4240065bd..bb72d0e15cbd 100644
--- a/arch/arm/boot/dts/st/stm32mp151.dtsi
+++ b/arch/arm/boot/dts/st/stm32mp151.dtsi
@@ -1681,6 +1681,14 @@ usbphyc_port1: usb-phy@1 {
 			};
 		};
 
+		ddrperfm: perf@5a007000 {
+			compatible = "st,stm32-ddr-pmu";
+			reg = <0x5a007000 0x400>;
+			clocks = <&rcc DDRPERFM>;
+			resets = <&rcc DDRPERFM_R>;
+			status = "disabled";
+		};
+
 		usart1: serial@5c000000 {
 			compatible = "st,stm32h7-uart";
 			reg = <0x5c000000 0x400>;
-- 
2.39.5

